pyat/at/lattice/utils.py

```diff
@@ -11,6 +11,14 @@
 import itertools
 
 
+class AtError(Exception):
```

Comment:
Don't these belong in the `at` package?

Reply:
When I tried to import `at.physics` in `lattice_object`, I had an import error. To solve that, I had to modify imports in many modules. At the moment, in a module I don't import `at` anymore, neither single modules, but `at` sub-packages (`lattice`, `physics`, `tracking`). So I had to move `AtError` to the `lattice` package. It works now, but only with the proper order of imports in `at/__init__.py`. It must come from circular imports, I don't really understand why.

Reply:
In fact, it does not work in python 2.7:
`AttributeError: attribute '__doc__' of 'instancemethod' objects is not writable`

Reply:
It looks like there's a different way to do it in Python 2, so I've pushed a commit.